README.md

```diff
@@ -1 +1,129 @@
-# scalardb-samples
\ No newline at end of file
+# Scalar DB Samples
+- This repository contains the sample application that will connect to `ScalarDB Server` as backend. For using native `ScalarDB` library to connect to `ScalarDB` directly please refer to this document [`Getting Started`](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started.md).
+- `ScalarDB Server` is `gRPC` server that implements ScalarDB interface. More information about `ScalarDB Server` can be found [`here`](https://github.com/scalar-labs/scalardb/tree/master/docs/scalardb-server.md)
+
+## Our sample application
+A simple electronic money cli app which has features:
+- Charge an `amount` to a `user_id`
+- Pay `amount` from a `user_id` to another `user_id`
+
+## Prerequisites
+- Java (OpenJDK 8 or higher)
+- Gradle
+- Docker, Docker Compose
+
+## Setting up
+### ScalarDB Server
+- In this sample we will use Cassandra as storage for our ScalarDB server. The Cassandra will also run along with ScalarDB Server in containers. The setting for our ScalarDB Server is [`database.properties`](./database.properties)
+```
+# Comma separated contact points
+scalar.db.contact_points=cassandra
+
+# Port number for all the contact points. Default port number for each database is used if empty.
+scalar.db.contact_port=9042
+
+# Credential information to access the database
+scalar.db.username=cassandra
+scalar.db.password=cassandra
+
+# Storage implementation. Either cassandra or cosmos or dynamo or jdbc can be set. Default storage is cassandra.
+scalar.db.storage=cassandra
+```
+- To start our ScalarDB Server. We use below command (noted that we should wait around a bit more one minute to make sure every container is already up, because ScalarDB container have to wait for Cassandra already up)
```

Comment:
In addition to Toshi's suggestion.
```suggestion
- To start our ScalarDB Server. We use below command (noted that we should wait around a bit more one minute to make sure every container is already up, because ScalarDB container has to wait for Cassandra container to be fully started)
```